### PR TITLE
Prepare release v1.47.0

### PR DESCRIPTION
**Pull request by jkowall**
Release v1.47.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Changes by Version
 ==================
 
-next release (yyyy-mm-dd)
```
Comment (albertteoh) [inline CHANGELOG.md:4]: We'll want to keep the next release section, esp. for documenting breaking changes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,13 +13,33 @@ next release (yyyy-mm-dd)
...
+* [hotROD] Fix OTEL logging in HotRod example ([@albertteoh](https://github.com/albertteoh) in [#4556](https://github.com/jaegertracing/jaeger/pull/4556))
+* [hotrod] Reduce span exporter's batch timeout to let the spans be exported sooner ([@GLVSKiriti](https://github.com/GLVSKiriti) in [#4518](https://github.com/jaegertracing/jaeger/pull/4518))
+
+* Build improvement to rebuild jaeger-ui if the tree does not match any tag ([@bobrik](https://github.com/bobrik) in [#4553](https://github.com/jaegertracing/jaeger/pull/4553))
+* [Test] Fixed a race condition causing unit test failure by changing the logging  ([@yurishkuro](https://github.com/yurishkuro) in [#4546](https://github.com/jaegertracing/jaeger/pull/4546)) resolves [#4497](https://github.com/jaegertracing/jaeger/issues/4497)
+
+
 ### UI Changes
 
+* UI pinned to version [1.31.0](https://github.com/jaegertracing/jaeger-ui/blob/main/CHANGELOG.md#v1300-2023-06-05).
```
Comment (albertteoh) [inline CHANGELOG.md:41]: The URL needs updating.

Comment (jkowall) [reply CHANGELOG.md]: Addressed in last push

File: CHANGELOG.md (hunk 1 above)
Comment (jkowall) [reply CHANGELOG.md]: Addressed in last push

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,19 +7,51 @@ next release (yyyy-mm-dd)
...
+#### ⛔ Breaking Changes
+
 * [SPM] Due to a breaking change in OpenTelemetry's prometheus exporter ([details](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.80.0))
   metric names will no longer be normalized by default, meaning that the expected metric names would be `calls` and
   `duration_*`. Backwards compatibility with older OpenTelemetry Collector versions can be achieved through the following flags:
   * `prometheus.query.normalize-calls`: If true, normalizes the "calls" metric name. e.g. "calls_total".
   * `prometheus.query.normalize-duration`: If true, normalizes the "duration" metric name to include the duration units. e.g. "duration_milliseconds_bucket".
 
-
 #### New Features
```
Comment (yurishkuro) [inline CHANGELOG.md:28]: I would not consider any of these as new features, rather minor improvements.

Comment (jkowall) [reply CHANGELOG.md:28]: Okay, when I look at a feature this is something the user can take advantage of directly, so for me at least the tracegen one is a new feature. I will revise as you wish.

Comment (yurishkuro) [reply CHANGELOG.md:28]: agree about tracegen, but the hotrod changes are mostly "internal". So it `Configuration.Close()` next item, not something that would affect a regular user.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,19 +7,47 @@ next release (yyyy-mm-dd)
...
+* [Cassandra] Add Configuration.Close() to ensure TLS cert watcher is closed ([@kennyaz](https://github.com/kennyaz) in [#4515](https://github.com/jaegertracing/jaeger/pull/4515))
+* Add *.kerberos.disable-fast-negotiation option to Kafka consumer ([@pmuls99](https://github.com/pmuls99) in [#4520](https://github.com/jaegertracing/jaeger/pull/4520))
 
 #### Bug fixes, Minor Improvements
 
+* Support normalized metric names ([@albertteoh](https://github.com/albertteoh) in [#4555](https://github.com/jaegertracing/jaeger/pull/4555))
```
Comment (yurishkuro) [inline CHANGELOG.md:34]: this one does sound like new feature, you even mention it in breaking changes. I would make the caption more descriptive.

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,19 +7,51 @@ next release (yyyy-mm-dd)
...
+* [hotROD] Replace gRPC instrumentation with OTEL ([@afzal442](https://github.com/afzal442) in [#4558](https://github.com/jaegertracing/jaeger/pull/4558))
+* [hotROD]: Upgrade `redis` service to use native OTEL instrumentation ([@afzal442](https://github.com/afzal442) in [#4533](https://github.com/jaegertracing/jaeger/pull/4533))
+* [tracegen] Add options to generate more spans and attributes for additional use cases ([@yurishkuro](https://github.com/yurishkuro) in [#4535](https://github.com/jaegertracing/jaeger/pull/4535))
+* Pass a wrapper instead of `opentracing.Tracer` to ease migration to OTEL in the future [part 1] ([@afzalbin64](https://github.com/afzalbin64) in [#4529](https://github.com/jaegertracing/jaeger/pull/4529))
 
 #### Bug fixes, Minor Improvements
 
+* [Cassandra] Add Configuration.Close() to ensure TLS cert watcher is closed ([@kennyaz](https://github.com/kennyaz) in [#4515](https://github.com/jaegertracing/jaeger/pull/4515))
+* Add *.kerberos.disable-fast-negotiation option to Kafka consumer ([@pmuls99](https://github.com/pmuls99) in [#4520](https://github.com/jaegertracing/jaeger/pull/4520))
```
[the remark this answers is not in the document]
Comment (jkowall) [reply CHANGELOG.md:30]: Adjusted this one, should be fixed now.

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,19 +7,51 @@ next release (yyyy-mm-dd)
...
+
+* Support normalized metric names ([@albertteoh](https://github.com/albertteoh) in [#4555](https://github.com/jaegertracing/jaeger/pull/4555))
+
```
[the remark this answers is not in the document]
Comment (jkowall) [reply CHANGELOG.md:34]: Removed them, was trying to group things to make it more readable. I could have added sections, but I noticed that was not done in the other release notes.

File: CHANGELOG.md [hunk 7]
```diff
@@ -7,19 +7,51 @@ next release (yyyy-mm-dd)
...
+* Add readme for memstore plugin ([@yurishkuro](https://github.com/yurishkuro) in [283bdd9](https://github.com/jaegertracing/jaeger/commit/283bdd93cbb4a467842625d8eb320722fcb83494))
+
+* [hotROD] Fix OTEL logging in HotRod example ([@albertteoh](https://github.com/albertteoh) in [#4556](https://github.com/jaegertracing/jaeger/pull/4556))
```
[the remark this answers is not in the document]
Comment (jkowall) [reply CHANGELOG.md:40]: Done

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**